src/redux/reducers/differentialExpression/differentialExpressionUnselectDeletedOption.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-param-reassign */
+import produce from 'immer';
+import { getCellSetKey } from 'utils/cellSets';
+
+const differentialExpressionUnselectDeletedOption = produce((draft, action) => {
+  const { key: deletedKey } = action.payload;
+
+  Object.keys(draft.comparison.group).forEach((comparisonType) => {
+    const comparisonGroup = draft.comparison.group[comparisonType];
```

Comment:
Does this work? usually I can't pick up stuff from `draft` without first doing a `current` or `original`

Reply:
Yea, it seems to work... I see the intended outcome. I'll check if there is a difference between using `current()` or direct access like this.

Reply:
So it seems that `current()` has to be used on a `draft`, which is a proxy to an object. When you directly access a value that is not an object, accessing draft directly returns the value, so it's comparable. If you try to access `current()` on a primtive value (e.g. not-object), it'll throw an error.

In this case, I'm accessing a property which is a primitive(string), so it returns the real value which can then be used to compare.